test/integration/connect/envoy/Dockerfile-tcpdump

```diff
@@ -0,0 +1,7 @@
+FROM alpine:latest
+
+RUN apk add --no-cache tcpdump
```

Comment:
I got tired of trying to figure out which requests were going where based off envoy. So I made it so that we can get packet captures from either the primary network namespace or the secondaries. It was so much more helpful than trying to figure out request flow from Envoy logs.